trin-core/src/portalnet/overlay_service.rs

```diff
@@ -939,6 +941,14 @@ where
 
     /// Processes an incoming request from some source node.
     fn process_incoming_request(&mut self, request: Request, _id: RequestId, source: NodeId) {
+        match request {
+            Request::Ping(ping) => self.process_ping(ping, source),
+            _ => {}
+        }
+    }
+
+    /// Add source NodeId to overlay routing table or update its connection state
+    fn update_routing_table(&mut self, source: NodeId) {
```

Comment:
This function name is quite vague, so if you see it being called somewhere, then it's not clear what it's doing. A more specific name would improve readability. We already have a function named `connect_node`, so we cannot use that. This function is sort of like a "touch" because we also want to update the node's position in its k-bucket if it's already present.

That said, I don't feel great about `touch_node` either, but hopefully that idea suggests a more useful name.

Reply:
I like `register_node_activity` most :1st_place_medal:.